src/NServiceBus.Metrics/MetricsFeature.cs

```diff
@@ -15,50 +14,15 @@ protected override void Setup(FeatureConfigurationContext context)
 
         var settings = context.Settings;
         var options = settings.Get<MetricsOptions>();
-        var endpointName = settings.EndpointName();
 
         SetUpRegisteredObservers(context, options, probeContext);
-
-        SetUpLegacyReporters(context, options, endpointName, probeContext);
     }
 
     static void SetUpRegisteredObservers(FeatureConfigurationContext context, MetricsOptions options, ProbeContext probeContext)
```

Comment:
Can we inline this method. I'm not sure it brings value now when it's the only one called in feature `Setup`